tests/core/test_siglongjmp.c

```diff
@@ -0,0 +1,19 @@
+#include <setjmp.h>
+#include <stdio.h>
+
+int main() {
+    sigjmp_buf env;
```

Comment:
Could you add a comment `// TODO: Once signals are supported, improve this test to verify preservation of signals state.`